api/src/main/java/org/apache/cloudstack/api/command/user/vm/ListVMsCmd.java

```diff
@@ -98,7 +99,7 @@ public class ListVMsCmd extends BaseListRetrieveOnlyResourceCountCmd implements
                collectionType = CommandType.STRING,
                description = "comma separated list of vm details requested, "
                    + "value can be a list of [all, group, nics, stats, secgrp, tmpl, servoff, diskoff, backoff, iso, volume, min, affgrp]."
-                   + " If no parameter is passed in, the details will be defaulted to all")
+                   + " If no parameter is passed in, the details will be defaulted to all.")
```

Comment:
one more - could we mention in the API doc string here, about the global setting that determines whether stats will be sent or not by the list API?